src/utils.js

```diff
@@ -41,6 +45,84 @@ const stringifyBuffRec = (obj) => {
   return obj
 }
 
+/**
+ * Downloads url and saves to tmp file
+ * @returns {filePath}
+ */
+export const downloadFile = async (downloadUrl) => {
+  console.log('Downloading url: ', downloadUrl)
```

Comment:
this log is duplicated at the call site. Consider removing it here and just leaving it there.